.github/workflows/ci.yaml

```diff
@@ -244,15 +239,24 @@ jobs:
       - name: Install Redis Operator
         run: |
           make deploy IMG=redis-operator:e2e
+          sleep 10
```

Comment:
instead of sleep i would prefer to use the 
```
kubectl wait 
```
as we have done below

Reply:
it will be removed after debug :)